.gitlab-ci.yml

```diff
@@ -968,24 +968,29 @@ latest_release:
 
 dca_tag_release:
   <<: *docker_tag_job_definition
-  <<: *run_when_triggered_on_tag
+  only:
+    refs:
+      - tags
```

Comment:
I this you can. Look at those 2 doc https://docs.gitlab.com/ee/ci/yaml/#only-and-except-simplified and https://docs.gitlab.com/ee/ci/variables/#predefined-variables-environment-variables.

I'm guessing this will do the trick:
```yaml
only:
  refs:
    - tags
  variables:
    - CI_COMMIT_TAG =~ /^dca([\d.-]|rc)+$/
```

I didn't tested it ...